contents/newsletter/product-pricing-advice.md

```diff
@@ -0,0 +1,175 @@
+
+- **Figma** launched an [enterprise plan in 2022](https://www.figma.com/blog/design-needs-everyone-new-plans-for-companies-that-design-together/) with more admin tools, workspaces, and access controls. They also [raised prices](https://www.figma.com/blog/billing-experience-update-2025/) for Figma Design in 2025.
+
+- **Intercom** reinvigorated and [doubled its growth](https://lennysvault.com/insights/growth-scaling-tactics/fba11d05-3cf2-4591-a811-995a1fcb325c) by launching its AI agent, Fin, with outcomes-based pricing.
+
+![Fin pricing](https://res.cloudinary.com/dmukukwp6/image/upload/w_1600,c_limit,q_auto,f_auto/Clean_Shot_2025_10_13_at_16_41_562x_fd6884de36.png)
+
+Changing your pricing enables you to better align the value you create with what you charge for it. For example, our [data pipelines product](/customer-data-infrastructure/destinations) used to be charged based on events ingested, but then [moved to rows exported and triggered events](/blog/data-pipeline-pricing) to better align what users actually used.
+
+The important part of doing this is making sure you communicate the change well. Our tips for doing this include:
+
+- Communicating early. Figma announced pricing changes in December 2024 that took effect March 2025.
+
+- Give people a warning and a discount (grace period) if you are going to raise their prices.
+
+- Understand who is going to pay more or less. 98% of our data pipeline customers paid less, but for the 2% who paid more, we made sure to communicate why we did this and help lower the impact of the change.
+
+- We try to grandfather existing customers’ pricing unless it is more expensive or we have a very good reason not to.
+
+## 3. How you price is who you are
+
+Imagine two [feature flag](/newsletter/feature-flag-mistakes) tools, they charge by:
+
+1. Seats and flags created.
+2. Requests. 
+
+Charging by seats and flags created will likely appeal to large enterprises who ship relatively few features. Enterprise features like auditability and access controls will likely be the focus.
+
+Charging by requests means more developers can access and create flags, but might be more sensitive about where they put those flags in their app so they aren’t charged too much for requests.
+
+These show the impact pricing can have on the overall product experience. **How much** you charge often doesn’t matter as much as **how** you charge and there are many ways this can vary like:
+
+- Usage-based pricing vs. seat-based pricing.
+- Pay-as-you-go vs. annual contracts.
+- Self-serve vs. custom contracts with sales.
+
+How you decide this likely comes down to your [ideal customer](/newsletter/ideal-customer-profile-framework). For us, that is [product engineers](/blog/what-is-a-product-engineer) at high-growth startups. The ability to self-serve and have usage-based pricing is important for them, so that is what we’ve gone with.
+
+## 4. Know the three Cs of pricing
```

Comment:
A few thoughts about this section:

- It doesn't really work with the headline because it's not specific enough. You could probably make all three of these points separately, rather than trying to combine them.

- How many of these are really non-obvious? If we had to pick one to go deeper on, which would it be?